client/src/js/actions/messages.js

```diff
@@ -0,0 +1,17 @@
+// For the sake of simplicity, we'll always get ALL the messages.
+export const UPDATE_MESSAGES = 'UPDATE_MESSAGES';
+export function updateMessages(messages) {
+  return {
+    type: UPDATE_MESSAGES,
+    messages
+  };
+}
+
+// This action updates a users message. Won't work (backend check) if the user does not have a valid JWT
+export function updateMessage(text) {
+  return (_, __, socket) => {
```

Comment:
@alexander-daniel Why obfuscate an argument using `_`?

Reply:
Just to avoid having unused variables, and to cut down on visual clutter. Maybe not the best approach, but since I'm only using the third parameter here, that's the only one I really care about naming.

Agreed though that this is perhaps not the most easily grok'able way of doing things. 

In my case, it's a way to get my linter to not complain about unused variables 😎